src/Symfony/Bridge/ProxyManager/LazyProxy/PhpDumper/ProxyDumper.php

```diff
@@ -65,19 +65,20 @@ public function isProxyCandidate(Definition $definition)
     /**
      * {@inheritdoc}
      */
-    public function getProxyFactoryCode(Definition $definition, $id, $methodName = null)
+    public function getProxyFactoryCode(Definition $definition, $id, $methodCall = null)
```

Comment:
I would name it `instantiationExpression` instead, as it may not even be a method call.